.github/workflows/discord-i18n-ping.yml

```
@@ -0,0 +1,35 @@
+name: Translation Tuesday
+
+on:
+  schedule:
+    - cron:  '0 0 * * TUE'
+  workflow_dispatch:
+
+jobs:
+  translation:
+    name: translation_tuesday
+    if: ${{ github.repository_owner == 'withastro' && github.event.commits[0].message != '[ci] format' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Tools & Dependencies
+        uses: ./.github/actions/install
+
+      - id: message
+        name: Format Discord message
+        env:
+          COMMIT_AUTHOR: ${{ github.event.commits[0].author.name }}
+          COMMIT_MESSAGE: ${{ github.event.commits[0].message }}
+          COMMIT_ID: ${{ github.event.commits[0].id }}
+        run: node ./scripts/tuesday-bot.mjs
```

```suggestion
        run: pnpm tsm --require=./scripts/lib/filter-warnings.cjs ./scripts/tuesday-bot.ts
```
We need to use this command as the script is using TypeScript.

---

Ah yep, this was from before I switched to typescript. Good catch!